modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/introspection/validation/ParameterModelValidator.java

```diff
@@ -191,4 +181,50 @@ private void validateNameCollisionWithTypes(ParameterModel parameterModel, Strin
 
     }
 
-}
+    private void validateParameterGroup(ParameterModel parameterModel)
+    {
+        ParameterGroupModelProperty parameterGroup = parameterModel
+                .getModelProperty(ParameterGroupModelProperty.class)
+                .orElse(null);
+
+        if (parameterGroup != null)
+        {
+            parameterGroup.getGroups().stream()
+                    .map(p -> p.getField())
+                    .forEach(field -> {
+                        if (!isInstantiable(field.getType(), false))
+                        {
+                            throw new IllegalParameterModelDefinitionException(
+                                    format("Parameter group type should be instantiable. Affected field [%s] in [%s].",
```

Comment:
rephase this to something more understandable. Say something like `an instantiable type with a default constructor was expected, however type %s was expected`